make/protogen.mk

```diff
@@ -58,7 +58,11 @@ PROTOC_FILE := $(PROTOC_DOWNLOADS_DIR)/$(PROTOC_ZIP)
 
 $(PROTOC_FILE): $(PROTOC_DOWNLOADS_DIR)
 	@echo "+ $@"
+ifeq (, $(shell which wget))
 	$(SILENT)wget -q "https://github.com/protocolbuffers/protobuf/releases/download/v$(PROTOC_VERSION)/$(PROTOC_ZIP)" -O "$@"
+else
+	$(SILENT)curl -sSL "https://github.com/protocolbuffers/protobuf/releases/download/v$(PROTOC_VERSION)/$(PROTOC_ZIP)" -o "$@"
+endif
```

Comment:
1. It makes little sense to have two ways of doing the same thing under a conditional. If you for any reason don't want to install `wget`, make `curl` command to be the only one here.
2. Please don't forget to add `--fail` command-line option to curl. Otherwise, it will happily download an error page.

Reply:
I am torn here, as I know `curl` much better than `wget` and I am sure that there are people that have totally opposite opinion. Let me install `wget` then